### ISSUE TITLE
[3.4.0] Position of "hints" in backend 

### ISSUE THREAD

**Issue by wangaz** [contao/core#7561]
When using the _flexible_ backend theme, the _hints_ (arrows) are not properly located:
![Hints](https://cloud.githubusercontent.com/assets/7368945/5691941/17582ab2-98df-11e4-9668-da5baf90c63f.png)

**Comment by Evgenioni** [comment]
There is a problem with hints, they function strange in flexible theme:
![2](https://cloud.githubusercontent.com/assets/2331446/5753999/8d7f659e-9cbc-11e4-96b9-abbef5786570.gif)

Also, there are problems with input highlight:
![1](https://cloud.githubusercontent.com/assets/2331446/5753962/e0c81c10-9cbb-11e4-9a25-3aba376ad35d.gif)

I use this code:
```css
.tl_text_trbl,.tl_imageSize_0,.tl_imageSize_1,#ctrl_gradient input,#ctrl_playerSize input {
	background-image:url("images/hints.gif");
	background-repeat: no-repeat;
	background-position: -20px 20px;
}
#ctrl_playerSize_0 {
	background-position:99.5% 1px !important;
}
#ctrl_playerSize_1 {
	background-position:99.5% -18px !important;
}
.tl_imageSize_0 {
	background-position:99.5% 1px !important;
}
.tl_imageSize_1 {
	background-position:99.5% -18px !important;
}
.trbl_top {
	background-position:99.5% -39px !important;
}
.trbl_right {
	background-position:99.5% -59px !important;
}
.trbl_bottom {
	background-position:99.5% -79px !important;
}
.trbl_left {
	background-position:99.5% -99px !important;
}
#ctrl_shadowsize_top {
	background-position:99.5% -119px !important;
}
#ctrl_shadowsize_right {
	background-position:99.5% -139px !important;
}
#ctrl_shadowsize_bottom {
	background-position:99.5% -159px !important;
}
#ctrl_shadowsize_left {
	background-position:99.5% -179px !important;
}
#ctrl_borderradius_top {
	background-position:0 -199px !important;
}
#ctrl_borderradius_right {
	background-position:99.5% -219px !important;
}
#ctrl_borderradius_bottom {
	background-position:99.5% -241px !important;
}
#ctrl_borderradius_left {
	background-position:0 -261px !important;
}
```
and
```css
/* Wide screen */
@media (min-width:1200px) {
...
	#ctrl_playerSize_0 {
		background-position:189px 1px !important;
	}
	#ctrl_playerSize_1 {
		background-position:190px -18px !important;
	}
```
in

**Comment by xchs** [comment]
You must first select an option from the resize mode select field.

**Comment by Aybee** [comment]
We can change this
```css
#ctrl_playerSize_0 {
  background-position: 164px 1px !important;
}
```
to this
```css
#ctrl_playerSize_0 {
  background-position: right 1px !important;
}
```
If the symbol then is to close on the right edge, we can make /system/themes/flexible/images/hints.gif about 2px wider.

**Comment by leofeyer** [comment]
Fixed in 9f8c3278ce73cfc3f416597e1a3dd49f57bc3510.
